tensornetwork/node.py

```diff
@@ -115,7 +115,7 @@ def get_tensor(self):
   def set_tensor(self, tensor):
     self.tensor = tensor
 
-  def reorder_edges(self, edge_order: List["Edge"]) -> "Node":
+  def reorder_edges(self, edge_order: List["edge.Edge"]) -> "Node":
```

Comment:
Think the rename was a little over-zealous. The strings should not be altered.

Reply:
I think all of the strings just need to be removed. idk, the circular dependence of `Edge` and `Node` might cause confusing issues with pytype. I think this refactor might've been too hard for a contributor. :/

Reply:
Oh, I misread this anyway. Didn't realize these were types! Please disregard my comment.

Reply:
It doesn't yet. @kshithijiyer needs to merge, so the tests haven't run.